core/lib/file-storage.js

```diff
@@ -0,0 +1,201 @@
+// Taken from https://github.com/VeliovGroup/Meteor-Files/blob/master/docs/aws-s3-integration.md
+import { Meteor } from 'meteor/meteor';
+import { _ } from 'meteor/underscore';
+import { Random } from 'meteor/random';
+import { FilesCollection } from 'meteor/ostrio:files';
```

Comment:
Don't we need to add a dep for this one as well?

Reply:
The package was already in: https://github.com/l3mpire/lemverse/blob/19a80c88c939892a865bc2f82d0182ec944c8e66/app/.meteor/packages#L35

Yet some files have been removed during my rebase. I'll fix it ASAP.